app/views/products/new.html.haml

```diff
@@ -6,5 +6,5 @@
   = form.text_field :title, placeholder: 'title'
   = form.text_field :description, placeholder: 'description'
   = form.number_field :price, placeholder: 'price'
-  = form.file_field :picture
+  = form.file_field :picture, direct_upload: true
   = form.submit
```

Comment:
Also, for better readability, you can group the related fields by leaving empty lines around them, like so:
`
= form.text_field :title, placeholder: 'title'  

= form.text_field :description, placeholder: 'description'  

= form.number_field :price, placeholder: 'price'  

= form.file_field :picture  
= form.file_field :picture, direct_upload: true  

= form.submit`